### DISCUSSION TITLE
shortcut with generic Google query : open each result in a new window

### DISCUSSION THREAD

**Discussion by luutuankiet** [Ulauncher/Ulauncher#1228] [Q&A]
### Description

hi, I am looking for a way to config the following `shortcut` set as my default search result.
Ulauncher will take the input, open default browser (firefox) with the query below and pass on the input to `%s`. 
```
https://google.com/search?q=%s
```
**current behaviour** : if there are already a firefox instance opened, a new tab will be opened to pass the query.
**expected behaviour** : would be great if i can modify it to **open a new firefox windows** with each query execution. currently I don't see any option to modify that behaviour - or it could be as a script that I am not aware of?

for more context behind this suggestion, my workflows are mainly on webapps / browser which I find it easier to manage each of them as single windows, with support from this ulauncher extenstion [windows switcher](https://github.com/beajeanm/ulauncher-windows-switcher). Opening new tabs hence breaks this workflow.

### Communication guidelines

- [X] I have read and agree with Ulauncher's communication guidelines and checked the [existing enhancement requests](https://github.com/Ulauncher/Ulauncher/labels/enhancement) before suggesting this.

### Ulauncher v6

- [X] I have checked and couldn't find my suggestion in v6.

**Comment by friday** [comment]
If you find out how to do it in a script for your desktop environment, then you can make a shell script shortcut of it instead.
 
There's no generic way that works for every desktop environment and browser, so we can't support this via some setting. Maybe showing a list of options https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#extra-actions some time in the future though.
